ONSdigital/eq-survey-runner
# Secure jenkins deployment

### Changes

Allows jenkins to decrypt the public/private key pairs before deploying them to elastic beanstalk.
### How to test

Browse to the prod jenkins and trigger a deployment. In the console you should see the output of the scripts (it's currently setup to use this branch)
### Who can test

Anyone apart from @warrenbailey
